app/src/main/res/values-uk/strings.xml

```diff
@@ -14,7 +14,7 @@
     <string name="folder_view">Перемкнути у перегляд за теками</string>
     <string name="other_folder">Інша тека</string>
     <string name="show_on_map">Показати на мапі</string>
-    <string name="unknown_location">Невідоме розташування</string>
+    <string name="unknown_location">Невідоме місцерозташування</string>
     <string name="increase_column_count">Збільшити кількість колонок</string>
```

Comment:
-розташування - eng. location; 
-місцерозташування - eng. place location (a direct translation from Russian, typically is used in Russian as 'месторазмещение'; did not find a proper recommended option in dictionaries for Ukrainian; for me personally sounds like a 'buttery butter')

Reply:
Розташування — arrangement, disposition, position.
Місцерозташування (or, in other case and sense, місце розташування) — situation, location, locus.
Place location — direct translation of two separate words (місце and розташування) incorrectly combined by translation software into one expression.
One shouldn't translate it directly, but always take the general sense into account. Also we're not speaking about Russian here. Russian has its own language rules which may and obviously will be different from Ukrainian.

Reply:
Ukrainian dictionaries do not contain 'місцерозташування' (http://sum.in.ua/s/misceroztashuvannja)
A Russian dictionary contains 'местоположение' (https://www.vedu.ru/expdic/15429/) and similar (https://www.vedu.ru/expdic/m112/).
I am talking here about a not accepted translation of the word 'location' in Ukrainian language, however accepted in Russian. Please check the links to dictionaries.
How about to translate the word 'location' as 'розміщення' which has a root 'місце', i. e. 'location'?

Reply:
Also https://www.dict.com/ukrainian-english/location

Reply:
> Ukrainian dictionaries do not contain 'місцерозташування'

Seems like we refer to different vocabularies. Personally I did find a translation of 'місцерозташування' to English:
1. https://www.m-translate.com.ua/uk-en/word-misceroztasuvanna-translation#
2. https://translate.google.com.ua/?hl=uk&tab=wT1#view=home&op=translate&sl=uk&tl=en&text=місцерозташування
> How about to translate the word 'location' as 'розміщення' which has a root 'місце', i. e. 'location'?

While Ukrainian still doesn't have common-used translations of some English words, I'll gladly discuss and agree on some acceptable translation of each of these words.
Though I don't think this translation is 100% correct, in this particular case it may be acceptable for an average person to see 'розташування' it this string. Just to come to a mutual understanding with you.
Also 'Невідоме розміщення' doesn't seem to be better translation than 'Невідоме розтішування'.

Reply:
Hi, I agree to 'розташування'.
I checked the links for 'місцерозташування' you provided. But I think the word itself is a bit overinformative. The first part indicates a place, as well as the second part 'розташування' bares a meaning of an action or a condition related to a word 'місце', i.e. a place, a location (ukr. http://sum.in.ua/s/roztashuvannja, first and second paragraphs). So 'розміщувати', 'розміщуватися'  were used to describe the 'розташовувати', 'розташуватися'. That was the logic.
Having said that I realize that the the dictionary (http://sum.in.ua/) was created in 1970s, when many words and their meanings were copied from Russian. Currently many professional linguists try to use Ukrainian dictionaries from 1920s, 1930s as the starting point. 
Yesterday I found some sites which may be interesting for you:
 https://slovotvir.org.ua/ - a place for discussions on the use of foreign words in Ukrainian,
https://e2u.org.ua/ - a number of Ukrainian-English modern dictionaries on one site,
https://r2u.org.ua/ - a number of Ukrainian-Russian dictionaries on one site.
There are lots of discussions on the respective forums as well.